docs.bruno/Testing - Bruno.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Testing - Bruno
+  type: http
+  seq: 1
+}
+
+get {
+  url: https://api.github.com/users/usebruno
```

Comment:
Consider adding error handling for the HTTP request. This could include checking the response status code and handling any potential errors appropriately. [important]